jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -301,13 +301,12 @@ static void afterClinitInitialization() {
 		if (null == com.ibm.oti.vm.VM.getVMLangAccess()) {
 			com.ibm.oti.vm.VM.setVMLangAccess(new VMAccess());
 		}
+		/*[PR CMVC 150472] sun.misc.SharedSecrets needs access to java.lang. */
```

Comment:
I think we can remove this obsolete CMVC reference.

Reply:
Removed.